src/server/main_service.cc

```diff
@@ -72,6 +72,16 @@ ABSL_FLAG(bool, admin_nopass, false,
           "If set, would enable open admin access to console on the assigned port, without auth "
           "token needed.");
 
+ABSL_FLAG(string, replicaof, "",
+          "Empty by default. If not empty - specifies an IP and a port which "
```

Comment:
I'd slightly modify the text here. First, I don't think you need to specify the fact that it's empty by default, that's already printed with `--helpfull` and can be seen here.
Also, while this flag supports IPs, I imagine it also support hostnames (which are resolved via DNS). I wouldn't even explicitly say the target is Dragonfly, as we also support replicating Redis.
As such, maybe something like:
"Specifies a target master to replicate. Format should be ..." (and include hostname in the format)